app/components/signup-steps/step-one.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { debounce } from '@ember/runloop';
+import { JOIN_DEBOUNCE_TIME } from '../../constants/join';
```

Comment:
Better naming here?
DO we have different debounce times for different functionality?
```suggestion
import { TIME_TO_DEBOUNCE } from '../../constants/join';
```

Reply:
No, JOIN_DEBOUNCE_TIME is only on join form